docs/api/qiskit-transpiler-service/0.3/_package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This 0.3 folder is simply moving the files we had before. You can skip over these files